iOS/PostEditor/PostEditorView.swift

```diff
@@ -1,6 +1,7 @@
 import SwiftUI
 
 struct PostEditorView: View {
+    private let bodyLineSpacing: CGFloat = 17 * 0.5
```

Comment:
Nitpick: I generally structure the view's properties as follows:

- Property wrappers not owned by the view (Environment, EnvironmentObject, ObservedObject)
- Two-way property bindings (Binding)
- Property wrappers owned by the view (State)
- (Private) properties (var and let)

Don't bother changing this though, because I'm going to do a big refactor of the PostEditorView file anyhow and these properties will get moved around anyhow. 🙂